test/assets/offscreenbuttons.html

```diff
@@ -29,8 +30,9 @@
 <button id=btn9>9</button>
 <button id=btn10>10</button>
 <script>
-window.addEventListener('DOMContentLoaded', () => {
```

Comment:
I wasn't around when this test was created, but I imagine it was written this way assuming the favicon warning would be logged before `DOMContentLoaded`. Now that we have a favicon, we no longer need to try and avoid the error message.